src/components/ChannelView.vue

```diff
@@ -94,25 +118,29 @@
 <script>
 import Vue from "vue";
 import Message from "./Message";
-var MessageClass = Vue.extend(Message);
+import VueMarkdown from "vue-markdown";
 import MessageInput from "./MessageInput";
 import axios from "axios";
 import { Picker } from "emoji-mart-vue";
 
+var MessageClass = Vue.extend(Message);
+Vue.use(VueMarkdown);
+
 //import paste from "../paste";
 
 export default {
   name: "channel",
   data() {
     return {
       id: "",
+      base: "/",
       messages: [],
+      unacked_messages: {},
       connected: false,
       socket: {},
-      user_id: "",
+      user_id: "p",
```

Comment:
What does this mean?

Reply:
User was hardcoded, this is fixed in commit 07809bf by fetching the user object from the server.